hudi-common/src/main/java/org/apache/hudi/keygen/constant/KeyGeneratorOptions.java

```diff
@@ -56,6 +56,13 @@
       .withDocumentation("Partition path field. Value to be used at the partitionPath component of HoodieKey. "
           + "Actual value ontained by invoking .toString()");
 
+  public static final ConfigProperty<String> KEYGENERATOR_CONSISTENT_LOGICAL_TIMESTAMP_ENABLED = ConfigProperty
+      .key("hoodie.datasource.write.keygenerator.consistent.logical.timestamp.enabled")
+      .defaultValue("false")
+      .withDocumentation("When set to true, consistent value will be generated for a logical timestamp type column, "
```

Comment:
Can we add an example here so that users know what to expect if not for enabling this config. example covering both row writer and non-writer path.

Reply:
Good point. Added an example.